views/openacademy_menu.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <!-- ACTIONS OPENING VIEWS ON MODELS -->
+    <record id="action_window_course" model="ir.actions.act_window">
+      <field name="name">openacademy.menu_course</field>
```

Comment:
Action names should be human-readable. Actually, they're the default menu name when not specified and they're translatable.

This also applies for the below action.